README.md

```diff
@@ -27,7 +25,7 @@ gem install berta
 From source:
 
 ```bash
-git clone git://github.com/dudoslav/berta.git
+git clone git@github.com:the-berta-project/berta.git
```

Comment:
Use the HTTPS URL, please. This is accessible only to developers/owners of the repository.